addons/search-sprites/search-bar.css

```diff
@@ -0,0 +1,15 @@
+#sa-sprite-search-box {
+    color: black;
```

Comment:
Are there any classes built in to scratch that can style this?

Reply:
@RedGuy12 just the color or the entire search box?

Reply:
no i dont mean remove `color: black` i mean use a different class to style everything

Reply:
I'm not sure if such a class exists. I'll try checking.

(also, removing `color: black` was intentional)

Reply:
also im not sure if the current style completely fits in with scratch

Reply:
but if it did, it would take up too much height

Reply:
like this:
<img width="414" alt="Screenshot 2021-12-29 at 10 37 10 AM" src="https://user-images.githubusercontent.com/83728060/147629257-fa48c2d3-f253-4e06-bae2-42d6d44db061.png">

Reply:
> like this: <img alt="Screenshot 2021-12-29 at 10 37 10 AM" width="414" src="https://user-images.githubusercontent.com/83728060/147629257-fa48c2d3-f253-4e06-bae2-42d6d44db061.png">

A bit too big

Reply:
You can probably use the input_input-form_l9eYg class.